minari/data_collector/data_collector.py

```diff
@@ -461,3 +379,105 @@ def close(self):
         # Close tmp_dataset.hdf5
         self._tmp_f.close()
         shutil.rmtree(self._tmp_dir.name)
+
+
+def clear_buffer(
```

Comment:
Can this and`add_to_episode_buffer`become a method of `EpisodeBuffer` (after making it a proper class)?